server/main.go

```diff
@@ -13,8 +14,10 @@ import (
 )
 
 func main() {
-	configfn := "default.conf"
-	data, err := ioutil.ReadFile(configfn)
+	configfn := flag.String("config", "default.conf", "location of configuration file")
```

Comment:
shall we use pkg
`github.com/spf13/cobra`
for flag parsing here also well for consistent?